service/app/controllers/usermanagement/validator/MigrationRejectRequestValidator.java

```diff
@@ -0,0 +1,45 @@
+package controllers.usermanagement.validator;
+
+
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.request.BaseRequestValidator;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.services.sso.SSOManager;
+import org.sunbird.services.sso.impl.KeyCloakServiceImpl;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+
+public class MigrationRejectRequestValidator extends BaseRequestValidator {
+
+    private Request request;
+    private String authToken;
+    private static List<String> mandatoryParamsList = new ArrayList<>(Arrays.asList(JsonKey.ACTION, JsonKey.USER_ID, JsonKey.USER_EXT_ID));
+    private static SSOManager ssoManager = new KeyCloakServiceImpl();
+
+    private MigrationRejectRequestValidator(Request request, String authToken) {
+        this.request = request;
+        this.authToken = authToken;
+    }
+
+    public static MigrationRejectRequestValidator getInstance(Request request, String authToken) {
+        return new MigrationRejectRequestValidator(request, authToken);
+    }
+
+    public void validate() {
+        validateAuthToken();
+    }
+
+    private void validateAuthToken() {
```

Comment:
Don't use this approach : We are always doing verification from Global.java one entry point and if not valid that will throw error. if valied then it will add userId into ctx object that value u can receive in controller.

Reply:
@manzarul done